app/src/utils/url.js

```diff
@@ -0,0 +1,17 @@
+import * as apiCfg from "../config/apiConfig";
+
+export function getQueryParam(queryUrl) {
+  const params = new URLSearchParams(queryUrl);
+  return {
+    queryStr: params.get(apiCfg.SERVICES.queryParamName),
+    page: parseInt(params.get(apiCfg.SERVICES.queryParamPage)),
+    limit: parseInt(params.get(apiCfg.SERVICES.queryParamLimit))
+  };
+}
+
+export function encodeQueryData(data) {
```

Comment:
interestingly enough I just wrote essentially this same fn in another project... there, I did it functionally rather than explicitly iteratively. honestly, I think both read straightforwardly enough so I'm not suggesting a change, just thought you'd be curious to see it:

```
const encodeParams = params => {
  let paramStr = Object.keys(params).map((key) => {
    return encodeURIComponent(key) + '=' + encodeURIComponent(params[key])
  }).join('&');
  //use falseyness of empty string created by join when params is empty
  if (paramStr){
    paramStr = '?' + paramStr
  }
  return paramStr;
};
```